plugin

```diff
@@ -301,6 +301,7 @@
   "wassy92x/lovelace-entities-btn-group",
   "wassy92x/lovelace-ha-dashboard",
   "wilsto/pool-monitor-card",
+  "zanac/temperature-heatmap-card",
```

Comment:
<img width="743" alt="2023-12-03_21-09-14" src="https://github.com/hacs/default/assets/15093472/588395f8-3e18-409c-bbf6-bc4d178fac06">
It cannot be updated.